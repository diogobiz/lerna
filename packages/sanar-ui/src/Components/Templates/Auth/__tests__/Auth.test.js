import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESAuthTemplate from '../'

it('renders correctly', () => {
    const component = (
        <ESAuthTemplate>
            <h1>ESAuthTemplate</h1>
        </ESAuthTemplate>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
