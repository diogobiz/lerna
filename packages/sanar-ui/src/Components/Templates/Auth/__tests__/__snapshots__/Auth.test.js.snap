// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="es-auth-template"
>
  <div
    className="ant-row es-row"
    style={Object {}}
  >
    <div
      className="ant-col es-col ant-col-xs-24 ant-col-md-12"
      style={Object {}}
    >
      <div
        className="es-auth-template__input-data"
      >
        <div
          className="es-brand-header es-auth-template__input-data__header"
        >
          <img
            alt="Sanar Logo in brand header."
            src="{}"
          />
        </div>
        <div
          className="es-auth-template__input-data__content"
        >
          <div
            className="es-auth-template__input-data__content__terms"
          />
        </div>
      </div>
    </div>
    <div
      className="ant-col es-col es-auth-template__marketing ant-col-xs-0 ant-col-md-12"
      style={
        Object {
          "backgroundImage": "url(undefined)",
        }
      }
    >
      <div
        className="es-auth-template__marketing__content mb-xl"
      >
        <div
          className="es-typography es-auth-template__marketing__content--title es-typography__heading es-typography__heading--h3"
          data-testid="es-typography"
        />
        <div
          className="es-typography es-typography--regular es-typography__heading es-typography__heading--h6"
          data-testid="es-typography"
        />
      </div>
    </div>
  </div>
</div>
`;
